### PR TITLE
pr testing on gerrit 3

### PR DESCRIPTION
**Pull request by zzzeek**
Change-Id: Idd3b75699947a487f19e1804d334caa4b5636a87

*** **S T O P ! ! !** ***

**TL;DR: don't submit a one liner "fix" for a code bug, show us your use case. We will fix it.**

Is this pull request fixing a bug in the code and not just a typo in the documentation?   If so, does your pull request include a **TEST CASE** ?   Or have you filed a proper bug report that includes a **TEST CASE** ?

*** **WE DO NOT ACCEPT ONE-LINER CODE-FIX PULL REQUESTS WITH NO TESTS.** ***  The SQLAlchemy project does **NOT** merge bug fixes that aren't verified through tests, since such unverified code **IS A BUG**.   If you would like us to write the tests, that is fine, please **FILE A BUG REPORT** which includes a **TEST SCRIPT**.   

The vast majority of "drive by" pull requests are incorrect solutions to an ill-defined problem.  Please don't assume that the one line that fixed your problem is the actual fix, **NO MATTER HOW TRIVIAL**.    And above all **HAVE A NICE DAY!!**

### CODE REVIEW

**Review by sqla-tester — COMMENTED** [review]
OK, this is **sqla-tester** setting up my work on behalf of **zzzeek** to try to get revision b714699afb310c418ab872d8637ed43d2ea41af3 of this pull request into gerrit so we can run tests and reviews and stuff

**Comment by sqla-tester** [comment]
New Gerrit review created for change b714699afb310c418ab872d8637ed43d2ea41af3: https://gerrit.sqlalchemy.org/c/sqlalchemy/testgerrit/+/2542

**Comment by zzzeek** [comment]
mm comments might not work, OK

File: README.rst [hunk 1]
```diff
@@ -11,3 +11,5 @@ These files are:
 also supports a, q, z
 
 gerrit 3
+
+this is a pr testing on gerrit 3
```
Comment (CaselIT) [inline README.rst:15]: does gh -> gerrit work?

Comment (CaselIT) [reply README.rst:15]: it does

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

again

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/testgerrit/+/2542

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

OK here's a live comment again

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/testgerrit/+/2542

**Review by sqla-tester — COMMENTED** [review]
**mike bayer** (zzzeek) wrote:

here's a "review"

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/testgerrit/+/2542

* README.rst (line 6): really?
* README.rst (line 9): do we want bar.py  here?

File: README.rst (hunk 1 above)
Comment (sqla-tester) [inline README.rst:15]: **mike bayer** (zzzeek) wrote:

comment on the actual line that changed.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/testgerrit/+/2542

**Review by sqla-tester — COMMENTED** [review]
**mike bayer** (zzzeek) wrote:

this is another quote unquote review

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/testgerrit/+/2542

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

OK this is another long paragraph where I'm copying what's in that other comment.  I'm dealing with the API a bit here internally, and the reason I am writing this long sentence is because I need to see what this version of gerrit sends to the API on the commandline when the comment is really long, is it just sending the first 25 characters, or what is it doing, and how am I going to match that to the gerrit API request, because while things seem a little better, they are still making us guess on how to find the comment that was submitted, there is even a nice "change id" they could give us now but they aren't.

here's a second paragraph.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/testgerrit/+/2542

**Comment by zzzeek** [comment]
OK i think it's good

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

test !

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/testgerrit/+/2542

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/c/sqlalchemy/testgerrit/+/2542 has been **abandoned**.  That means that at least for the moment I need to close this pull request. Sorry it didn't work out :(
